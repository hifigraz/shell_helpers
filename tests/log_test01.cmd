logInfo "ASDF %.2f %s" 2 !
