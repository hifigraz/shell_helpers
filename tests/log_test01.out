00000000:000000 INFO     ASDF 2.00 !
